eng/helix/helix.proj

```diff
@@ -37,16 +37,15 @@
       Windows.10.Amd64.ClientRS4.VS2017.Open;
       OSX.1012.Amd64.Open;
       Ubuntu.1810.Amd64.Open;
-      Ubuntu.1804.Arm64.Open;
       Ubuntu.1604.Amd64.Open;
```

Comment:
What about the other Ubuntu/Arm64 pairs?

Reply:
Same applies to Debian too.

Reply:
I think you might have misread (i did too the first time). Amd64 vs Arm64